File: test/fixtures/ecto/20080727191751_create_users.exs
defmodule MyApp.Repo.Migrations.CreateUsers do
  use Ecto.Migration
  def up do
    create table(:users) do
      add :created_at, :datetime
      add :updated_at, :datetime
      add :type, :string
      add :username, :string
      add :email, :string
      add :crypted_password, :string, size: 40
      add :salt, :string, size: 40
      add :destroyed_at, :datetime
      add :destroyed_by, :integer
    end
  end

  def down do
    drop table(:users)
  end
end

File: test/fixtures/20080727191751_create_users.rb
class CreateUsers < ActiveRecord::Migration
  def self.up
    create_table "users", :force => true do |t|
      t.datetime :created_at
      t.datetime :updated_at
      t.string   :type
      t.string   :username
      t.string   :email
      t.string   :crypted_password, :limit => 40
      t.string   :salt,             :limit => 40
      t.datetime :destroyed_at
      t.integer  :destroyed_by
    end
  end

  def self.down
    drop_table "users"
  end
end


defmodule Xdate.Repo.Migrations.CreatePayday do
  use Ecto.Migration

  def change do
    create table(:paydays) do
      add :exchange, :string
      add :ticker, :string
      add :own, :date
      add :x, :date
      add :pay, :date
      add :amount, :decimal

      timestamps
    end
  end
end
